frontend/public/locales/fr/renew-adult-child.json

```diff
@@ -353,46 +353,73 @@
         "dental-insurance-required": "Select whether the child has access to dental insurance"
       }
     },
-    "dental-benefits": {
-      "title-no-name": "Accès à d'autres prestations dentaires fédérales, provinciales ou territoriales",
-      "title": "{{childName}}\u00a0: accès à d'autres prestations dentaires fédérales, provinciales ou territoriales",
-      "access-to-dental": "L'accès à des prestations dentaires dans le cadre d'un programme social n'aura aucune incidence sur l'admissibilité de l'enfant au Régime canadien de soins dentaires.",
-      "select-one": "Sélectionnez une option",
-      "eligibility-criteria": "Si l'enfant répond à tous les critères d'admissibilité, sa couverture doit être coordonnée entre les régimes afin qu'il n'y ait pas de dédoublement ou de lacunes dans la couverture.",
+    "confirm-dental-benefits": {
+      "title": "(FR) {{childName}}: access to other federal, provincial or territorial dental benefits",
+      "access-to-dental": "(FR) Access to dental coverage through a provincial, territorial or federal government social programs will not impact the child's eligibility for the Canadian Dental Care Plan.",
+      "select-one": "(FR) Select one",
+      "eligibility-criteria": "(FR) If the child meets all the eligibility criteria, their coverage will be coordinated between the plans to ensure there are no gaps or duplication in coverage.",
```

Comment:
This PR replaces French text with placeholders, which I believe should not be the case.

Tagging @nibivi77 as he has been working on these French translations. I am not sure if the English i18n file is using old text or not.